### PR TITLE
Add more information on how to become a committer

### PR DESCRIPTION
**Pull request by tdcmeehan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,7 +34,9 @@ We recommend you use IntelliJ as your IDE. The code style template for the proje
...
+
+New committers are approved by majority vote of the TSC ([see TSC charter](https://github.com/prestodb/tsc/blob/master/CHARTER.md)).  To become a committer, reach out to an [existing TSC member](https://github.com/prestodb/tsc#members) and ask for their feedback on your eligibility (see: [How to become a Presto Committer?](https://github.com/prestodb/presto/wiki/How-to-become-a-Presto-committer%3F)).  Note: to expedite the process, consider creating a document that outlines your Github stats, such as the number of reviews, lines of code added, number of PRs, and outlines particularly outstanding code and review contributions.  If the TSC member believes you are eligible, they will submit your nomination to a vote, typically in the form of a PR that adds your handle to the `CODEOWNERS` file.  The process is complete once the PR is merged.
```
Comment (jaystarshot) [inline CONTRIBUTING.md:39]: Sorry I had a question, Is this majority vote of TSC of majority vote of committers?

Comment (tdcmeehan) [reply CONTRIBUTING.md:39]: It's a majority of TSC.  I will update it to make it clear.

**Review by xiaoxmeng — APPROVED** [review]
@tdcmeehan thanks!

**Review by majetideepak — APPROVED** [review]

**Review by steveburnett — APPROVED** [review]
LGTM!

**Review by yingsu00 — APPROVED** [review]

**Pull request merged**